.travis.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  - docker
+
+branches:
+  only:
+  - test
+ 
+script:
+  - ./build.sh
+  - ./deploy.sh -c docker -t
+  - ./deploy.sh -c docker -i kruize:$(cat .kruize-version) --timeout=60
+  - docker logs kruize | grep "CPU Limit"
```

Comment:
Looks like this will always pass. Can we do a `exit -1` if do not find the recommendations. Also grep for "Error" and fail if found.